### DISCUSSION TITLE
How to get the "typed" actor from the registry

### DISCUSSION THREAD

**Discussion by Darkskald** [slawlor/ractor#199] [Q&A]
The documentation states that in order to use an actor cell retrieved from the registry  "additionally requires knowledge of the [crate::Actor] in order to send messages". Assuming I know the type, how could I "downcast" or convert the ActorCell to the desired type? Could someone please provide a minimal example?

**Comment by slawlor** [comment]
```rust
let actor: ActorRef<SomeType> = cell.into();
```
